ansible_bender/utils.py

```diff
@@ -260,3 +260,48 @@ def is_ansibles_python_2(ap_exe: str) -> bool:
         return True
     logger.debug("it seems that %s is not using python 2", ap_exe)
     return False
+
+def fancy_time(build_time):
+    """ 
+    returns build time in human readable form 
+
+    :param build_time: total time taken for the build
+    """
+    microseconds = build_time.microseconds
+    milliseconds = round(microseconds / 1000)
+    seconds = build_time.seconds
+    minutes = round(seconds / 60)
+    hours = round(seconds / 60 / 60)
+    days = build_time.days
+    f_time = ""
+
+    if days > 0:
+        f_time = str(days)
+        f_time = f_time + (" day" if days == 1 else " days")
+    else if hours > 0:
+        f_time = str(hours)
+        f_time = f_time + (" hour" if hours == 1 else " hours")
+    else if minutes > 0:
+        f_time = str(minutes)
+        f_time = f_time + (" minute" if minutes == 1 else " minutes")
+    else if seconds > 0:
+        f_time = str(seconds)
+        f_time = f_time + (" second" if seconds == 1 else " seconds")
+    else if milliseconds > 0:
+        f_time = str(milliseconds)
+        f_time = f_time + (" millisecond" if milliseconds == 1 else " milliseconds")
+    else if microseconds > 0:
+        f_time = str(microseconds)
+        f_time = f_time + (" microsecond" if microseconds == 1 else " microseconds")
+    else:
+        raise Exception('Build time seems almost nothing.')
```

Comment:
why would you raise an exception when formatting time? A fallback value is a much better solution, e.g. `f_time = "now"`